### ISSUE TITLE
Kutt self-hosted support

### ISSUE THREAD

**Issue by sasakitz** [fwenzel/copy-shorturl#137]
I would like to see support for kutt in self-hostd as well as kutt.it.

**Comment by fwenzel** [comment]
That is probably not too hard, since I assume the API of kutt.it and a self-hosted instance is identical. However, I do not really have a way to test it?

**Comment by fwenzel** [comment]
Oh thank you this is super helpful. I have a setting for custom shortener domain in there but I think that's less important than the self-hosted domain type.

**Comment by sasakitz** [comment]
Support for self-hosting may be less important for the majority.
However, kutt.it is currently unable to accept new registrations and a reverse proxy is required to support SSL for custom domains.

**Comment by fwenzel** [comment]
Makes sense to me - the request for self-hosting is pretty common from Copy ShortURL users. I think I'll just add a new setting for it.
